src/lib/structures/errors.ml

```diff
@@ -95,7 +95,11 @@ exception Error of error
 
 let error e = raise (Error e)
 
-let typing_error e loc =
+let typing_error ?ae_loc e loc =
+  let () =
+    match ae_loc with
+    | None -> ()
+    | Some ae_loc -> Format.printf "Error at %s@." ae_loc in
```

Comment:
I agree but it was like this in this PR and I won't create a separated PR for such a tiny and inoffensive modification.

Reply:
I am not asking to make a separate PR for this change, I am asking that we remove it. This is clearly a temporary print that was added for debugging and shouldn't make it into the actual code, especially since it is not used at all.

Reply:
Ok :)